src/helpers/isSwitchingNetworkError.ts

```diff
@@ -0,0 +1,9 @@
+export default function (error: unknown) {
+  // Sometimes this error fires when user switching between networks too fast, we should not clean data in this case
+  if (
+    error instanceof Error &&
+    error.message.includes('Attempting to connect.')
```

Comment:
Hmm, what if some `axios` request will have such a message? Maybe we should change it?

Reply:
wdym? can you elaborate, please?

Reply:
nvm, I just removed this file